blog/_posts/2017-06-30-juliacon.md

```diff
@@ -0,0 +1,71 @@
+It’s a year later and I’m back: this time to the West Coast for the next edition of JuliaCon. 
```

Comment:
Needs a header:
```
---
layout: post
title:  West Coast is Best Coast for JuliaCon
author: Ranjan Arantharaman
---
```
(with the title replaced with whatever you'd like to call it)